ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -79,6 +79,7 @@ ethernet_interfaces:
         no_autoconfig_flag: true
     priority_flow_control:
       enabled: false
+    spanning_tree_guard: root
```

Comment:
please add test coverage of all input variants. If you test `none` before adjusting, you should see that it does not work, because Ansible will take it as the `None` value.

Reply:
good hint!!